deployment/docker-compose.ci.yml

```diff
@@ -5,4 +5,4 @@ services:
     build:
       context: ..
       dockerfile: ./deployment/Dockerfile
-    image: ${IMAGE_ROOT}/django:${GITHUB_SHA_SHORT}
+    image: ${REGISTRY}/${IMAGE_NAME}/django:${GITHUB_SHA_SHORT}
```

Comment:
Why do we need to change this?

Reply:
These environment variables are all defined in the workflow file (`deploy.yml` in this case). `IMAGE_ROOT` used to just have the registry hard-coded into it and then appended `IMAGE_NAME`. But I made `REGISTRY` a separate variable since it's used in a few places and got rid of `IMAGE_ROOT`, hence this change.